README.md

```diff
@@ -69,6 +69,38 @@ class NoteAdmin(admin.ModelAdmin):
 admin.site.register(Note, NoteAdminForm)
 ```
 
+## Configuration
+
+You can configure some of the editor behaviour by modifying the `DJANGO_TIPTAP_CONFIG` dict in `settings.py`.
+
+```python
+DJANGO_TIPTAP_CONFIG = {
+    "width": "500px",
+    "height": "500px",
+    "extensions": [
+        "bold",
+        "italic",
+        "underline",
+        "strikethrough",
+        "h1",
+        "h2",
+        "h3",
+        "h4",
+        "h5",
+        "h6",
+        "textAlign",
+        "indent",
+        "table",
+        "bulletList",
+        "orderedList",
+        "typography",
+        "clearFormat"
+    ],
+    "placeholderText": "Begin typing here...",  # set None to skip display
+    "unsavedChangesWarningText": "You have unsaved changes",  # set None to skip display
+}
+```
+
 # Contributing
 
 This project is a very rough draft of what a TipTap Editor experience in Django could
```

Comment:
Now that we provide the configs for having/not-having the extension, should we modify/remove this ?

Reply:
Yes, we should!